web/packages/teleport/src/Account/AccountNew.tsx

```diff
@@ -36,14 +36,33 @@ import AddDevice from './ManageDevices/AddDevice';
 import { ActionButton, Header } from './Header';
 import { PasswordBox } from './PasswordBox';
 
-export default function Container() {
+export interface EnterpriseComponentProps {
+  // TODO(bl-nero): Consider moving the notifications to its own store and
+  // unifying them between this screen and the unified resources screen.
```

Comment:
we already have a notificate store (we probably need to name this one a toaster store or name the other one something else, just fyi)

<img width="541" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/22a579f5-5bbf-4213-a985-299f4125ff84">

Reply:
@kimlisa Yeah, I saw it right after I typed that TODO. I think that this is a different case, though. If I remember correctly, the GCP console also uses this pattern: cross-page notifications and long running processes state use the bell icon, and short notifications, valid only for current page, are shown within the page itself.